### PR TITLE
python310Packages.flask-wtf: fix failing build

### PR DESCRIPTION
**Pull request by anthonyroussel**
###### Description of changes

* Fix failing build of flask-wtf
  * Caused by a failing test since Flask has been upgraded to 2.2
  * Disabled the failing test and we should wait for a fix to patch this issue upstream
  * I will create a PR soon on the Flask WTF repository to fix that: https://github.com/wtforms/flask-wtf/commit/ea0ba030d8f3e27c616ed666b4c618b15aea14e7

* Added myself to maintainers

Fixes the flask-wtf failing build on the staging-next PR/branch: https://github.com/NixOS/nixpkgs/pull/188284

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by anthonyroussel** [comment]
Pull request opened on the Flask WTF repository: https://github.com/wtforms/flask-wtf/pull/533

**Comment by anthonyroussel** [comment]
@ofborg build calibre-web

**Comment by anthonyroussel** [comment]
Result of `nixpkgs-review pr 190680` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages marked as broken and skipped:</summary>
  <ul>
    <li>apache-airflow</li>
    <li>python310Packages.apache-airflow</li>
    <li>python39Packages.apache-airflow</li>
  </ul>
</details>
<details>
  <summary>5 packages failed to build:</summary>
  <ul>
    <li>pgadmin4</li>
    <li>python310Packages.flask-admin</li>
    <li>python310Packages.ihatemoney</li>
    <li>python39Packages.flask-admin</li>
    <li>python39Packages.ihatemoney</li>
  </ul>
</details>
<details>
  <summary>10 packages built:</summary>
  <ul>
    <li>archivy</li>
    <li>calibre-web</li>
    <li>python310Packages.flask-appbuilder</li>
    <li>python310Packages.flask-mongoengine</li>
    <li>python310Packages.flask-security-too</li>
    <li>python310Packages.flask-wtf</li>
    <li>python39Packages.flask-appbuilder</li>
    <li>python39Packages.flask-mongoengine</li>
    <li>python39Packages.flask-security-too</li>
    <li>python39Packages.flask-wtf</li>
  </ul>
</details>

**Comment by anthonyroussel** [comment]
@ofborg build pgadmin4 python310Packages.flask-admin python310Packages.ihatemoney apache-airflow python310Packages.apache-airflow

**Comment by anthonyroussel** [comment]
pgadmin will be fixed with https://github.com/NixOS/nixpkgs/pull/188575

**Review by bhipple — APPROVED** [review]

**Comment by risicle** [comment]
Arguably you could just pull your fix as a patch, we may forget to ever remove the test disablement.

File: pkgs/development/python-modules/flask-wtf/default.nix [hunk 1]
```diff
@@ -32,10 +32,14 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  disabledTests = [
+    "test_set_default_message_language"
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/flask-wtf/default.nix:36]: Please add a comment why this test is disabled

Comment (anthonyroussel) [reply pkgs/development/python-modules/flask-wtf/default.nix]: > Arguably you could just pull your fix as a patch, we may forget to ever remove the test disablement.

Thanks for the review.
I removed this disabled test and use the fix as a patch in the package.

**Pull request merged**